### PR TITLE
tutor: add chapter for commenting lines

### PR DESCRIPTION
**Pull request by uncomfyhalomacro**
Hello. Just a small addition to the tutor. :D

### CODE REVIEW

File: runtime/tutor [hunk 1]
```diff
@@ -96,7 +96,7 @@ _________________________________________________________________
  2. Move to a place in the line which is missing text and type
     i to enter Insert mode. Keys you type will now type text.
  3. Enter the missing text.
- 4. Type Escape to exit Insert mode and return to Normal mode.
+ 4. type Escape to exit Insert mode and return to Normal mode.
```
Comment (michaeljones) [inline runtime/tutor:99]: I'm just a random person but I'm not sure why this has been lowercased? Seems like all the points start with a capital?

Comment (uncomfyhalomacro) [reply runtime/tutor:99]: Oh I didn't notice this!

**Comment by uncomfyhalomacro** [comment]
Okay. Done formatting and making the body the same number of lines as other sections. Thanks for the review! :heart:

**Comment by gabydd** [comment]
I have been working on #4718 it only changes the ``Ctrl-c`` binding a bit see my most recent comment but I think once this is merged we should add something about how you can block comment (``<space>cb``) is what it's binded to in the pr right now

**Comment by uncomfyhalomacro** [comment]
That's awesome. I guess we can just leave this PR open until that gets merged

**Comment by the-mikedavis** [comment]
We can take this as-is: it looks like the current behavior of C-c in #4718 on plaintext files is unchanged. We can revise this section with future changes once they're merged 👍

**Review by the-mikedavis — APPROVED** [review]
This looks great, thanks!

**Comment by gabydd** [comment]
> We can take this as-is: it looks like the current behavior of C-c in #4718 on plaintext files is unchanged. We can revise this section with future changes once they're merged 👍

That's what I was thinking as well I realize that probably wasn't clear in my comment

**Pull request merged**